Source/Core/DolphinWX/AuiToolBar.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <wx/aui/auibar.h>
+#include <wx/window.h>
+
+// XXX: This fixes wxAuiToolBar setting itself to 21 pixels wide regardless of content
```

Comment:
I'm pretty sure that "XXX" was only a search-marker to find those places when required. Since it's in our code now, you can drop that.